src/main/java/liquibase/ext/spanner/sqlgenerator/CreateSequenceGeneratorSpanner.java

```diff
@@ -36,12 +36,10 @@ public ValidationErrors validate(
     errors.checkDisallowedField("ordered", statement.getOrdered(), database, CloudSpanner.class);
     errors.checkDisallowedField("cycle", statement.getCycle(), database, CloudSpanner.class);
     errors.checkDisallowedField("dataType", statement.getDataType(), database, CloudSpanner.class);
-    // Allow setting incrementBy to 1.
-    if (!Objects.equals(BigInteger.ONE, statement.getIncrementBy())) {
-      errors.checkDisallowedField("incrementBy", statement.getIncrementBy(), database,
-          CloudSpanner.class);
-    }
-    
+    errors.checkDisallowedField("incrementBy", statement.getDataType(), database, CloudSpanner.class);
```

Comment:
This should be removed. See above, we should allow this value to be set to 1. Removing that now would be a breaking change.

Reply:
Removed

Reply:
how can it will be breaking change if support sequences was false?